README.md

```diff
@@ -0,0 +1,136 @@
+Please read the documentation available here:
+<http://mapfish.github.io/mapfish-print/>
+
+For support or questions post on the mailing list:
```

Comment:
Maybe on this occasion, it should be clarified what this mailing list is for. It is a community list, where users can share experiences, it is not a professional support mailing list (for professional support, users should sign a support contract with a professional services company, and use their appropriate channel).

Reply:
+1 :-)

Reply:
I will add this in the commit.